src/axom/primal/tests/primal_winding_number.cpp

```diff
@@ -168,9 +168,16 @@ TEST(primal_winding_number, closure_edge_cases)
 
   // Tests a potential issue where the query point is treated as being
   //  on the closure, but not on the edge of the approximating polygon.
-  EXPECT_NEAR(winding_number(Point2D({0, 1e-8}), quartic),
-              0.500000031830989,
-              abs_tol);
+  for(int i = 2; i < 15; ++i)
+  {
+    // In all cases, the winding number should be *near* 0.5.
+    //  If the tolerances don't match, we would get an "off-by-0.5" error
+
+    double diff = std::pow( 10, -i );
+    EXPECT_NEAR(winding_number(Point2D({0, diff}), quartic, 0.5 * diff, EPS), 0.5, 0.1);
+    EXPECT_NEAR(winding_number(Point2D({0, diff}), quartic, 1.0 * diff, EPS), 0.5, 0.1);
+    EXPECT_NEAR(winding_number(Point2D({0, diff}), quartic, 2.0 * diff, EPS), 0.5, 0.1);
+  }
```

Comment:
This is a great test!
Would it makes sense to also test `Point2D({0, -diff})` ?

Reply:
Probably! The internal tolerance checks don't care which side of the closure the point is on, but might as well check it here too.

Reply:
It actually made a lot of sense! There was the exact same issue with the `BoundingBox` check, since the closure is coincident with an edge. This means it would be counted as "outside" the bounding box but "on" the edge, and therefore give the wrong answer. I'm actually surprised that this didn't come up as an issue in the previous versions of `winding_number<Bezier>`, since nothing new to this PR has changed this behavior. Neat!